projects/front-end-library/src/lib/_doc/styles/1-colors/colors.js

```diff
@@ -32,14 +43,21 @@ const colors = [
                 name             : 'bg-highlight'
             },
             {
-                label            : 'Brand',
-                name             : 'brand'
+                label            : 'Overlay black 20%',
+                name             : 'bg-overlay-black-20prct',
+                hasUtilityClass  : false,
             },
             {
-                label            : 'Highlight',
-                name             : 'highlight',
+                label            : 'Overlay black 50%',
+                name             : 'bg-overlay-black-50prct',
                 hasUtilityClass  : false,
-            },
+            }
+        ]
+    },
+    {
+        name: 'Background (action)',
+        description: 'These background colors are meant to be used on interactive visual elements.',
```

Comment:
```suggestion
        description: 'Meant to be used on interactive visual elements.',
```
Pour aller droit au but.